ci/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+set -ex
+
+: "${TARGET?The TARGET environment variable must be set.}"
+
+FEATURES="rayon,serde"
+if [ "${TRAVIS_RUST_VERSION}" = "nightly" ]; then
+    FEATURES="${FEATURES},nightly"
+    export RUSTFLAGS="$RUSTFLAGS --cfg hashbrown_deny_warnings"
```

Comment:
This seems to be duplicated with the line below?

Also can't you just do "-D warnings" from the command-line?

Reply:
I did not know this was possible!